pkg/bb/cmd/main.go

```diff
@@ -19,9 +19,17 @@ func run() {
 	}
 }
 
+func isSymlink(f string) bool {
+	s, err := os.Stat(f)
+	if err != nil {
+		return false
+	}
+	return s.Mode()&os.ModeSymlink != 0
```

Comment:
actually, I would be a bit more careful here. While this is right now, there are cases where more than one bit is set to indicate a file type and it's burned people.
return (s.Mode()&os.ModeType) == os.ModeSymlink

this is similar to IsRegular()
I realize your form works however, I just feel safer with the full compare.